README.md

```diff
@@ -142,6 +142,58 @@ to `latest`:
 ORT_SERVER_IMAGE_PREFIX= ORT_SERVER_IMAGE_TAG=latest docker compose up
 ```
 
+#### Handling major version upgrades of PostgreSQL
+
+PostgreSQL does not support reading the data directory of previous major versions.
+When upgrading the PostgreSQL version in the Docker Compose setup, the database data directory must be manually
+migrated by following these steps:
+
+- Stop the Docker Compose setup:
+
+  ```shell
+  docker compose stop
+  ```
+
+- Start only the PostgreSQL service:
+
+  ```shell
+  docker compose up -d postgres
+  ```
+
+- Make a backup of the ORT Server and Keycloak databases:
+
+  ```shell
+  docker compose exec postgres pg_dump -Fc -U postgres -d ort_server -n public > keycloak.dump
```

Comment:
Side remark: IMO it's a bit weird that Keycloak does not use a more speaking schema than "public".

Reply:
I noticed that as well but didn't want to change it as part of this PR.